pkg/virt-launcher/virtwrap/network/podinterface_test.go

```diff
@@ -39,6 +39,22 @@ import (
 	"kubevirt.io/kubevirt/pkg/virt-launcher/virtwrap/api"
 )
 
+var testParams = map[string]struct {
+cidr    string
+samples map[string]string
```

Comment:
Why do the 'GW' and the 'VM' are inside a map. What is the difference between them and the 'cidr'?

Reply:
`Cidr` parameter represents the addresses range, while `VM` and `GW` are the VirtualMachine address and gateway address represented in CIDR notation (where you write the address and the range prefix explicitly)

Reply:
I meant, why  'GW' and 'VM' are inside a map while 'cidr' is not?

Reply:
I removed this map, now there is a struct with a property for each address/cidr